lib/src/chip/chip-row/chip-row.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, HostBinding, OnInit, Input } from '@angular/core';
+import { ChipComponent } from '../chip.component';
+import { FilterButtonComponent } from '../filter-button/filter-button.component';
+
+@Component({
+    selector: 'hc-chip-row',
+    templateUrl: './chip-row.component.html',
+    styleUrls: ['../chip.component.scss']
+})
+export class ChipRowComponent implements OnInit {
+    @HostBinding('class') hostClass = 'chip-row';
+    @Input() singleRow: boolean = false;
+    contentClass: string = 'chip-row-contents';
+
+    constructor( ) {
+    }
+
+    ngOnInit() {
+        if ( this.singleRow ) {
```

Comment:
You'll need to monitor the changing of singleRow so that you add/remove this single-row class. You can do this through the setter method for singleRow